contrib/syslog-debun

```diff
@@ -402,44 +449,87 @@ acquire_syslog_var () {
 	fi
 }
 
-acquire_syslog_ldinfo () {
-	ldd $syslogrealbin >${tmpdir}/syslog.ldd
+format_ldd_output () {
 	while read x ; do
 		x="${x#*=>}"
 		x="/${x#*/}"
 		x="${x%% (*}"
-		#[ "${x:0:2}" != "/ " ] && echo "$x" >>${tmpdir}/syslog.ldfiles
-		[ "${x}" != "/ " ] && echo "$x" >>${tmpdir}/syslog.ldfiles
-	done <${tmpdir}/syslog.ldd 
+		x="${x%%(*}"
+		[ "${x}" != "/ " ] && [ -f "${x}" ] && echo "$x"
```

Comment:
Nice rework. I assume why is this here, but not sure.
Would you mind add some example comment here showing what it transfrom to what?
Btw, thanks for this! I've just started to doubt in myself, because I have a suspicion, that ${x:0:2} is not in posix, so maybe a dash or ash can not interpret that brace expansion.